plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -5244,6 +5246,16 @@ protected Answer execute(ModifyStoragePoolCommand cmd) {
                 answer.setLocalDatastoreName(morDatastore.getValue());
             }
 
+            HostMO host = (HostMO) hyperHost;
+            boolean hardwareAccelerationSupportForDataStore = getHardwareAcceleationSupportForDataStore(host.getMor(), dsMo.getName());
+            StoragePoolInfo poolInfo = answer.getPoolInfo();
+            Map<String, String> poolDetails = poolInfo.getDetails();
+            if (poolDetails == null) {
+                poolDetails = new HashMap<>();
+            }
+            poolDetails.put("hardwareAccelerationSupported", String.valueOf(hardwareAccelerationSupportForDataStore));
+            poolInfo.setDetails(poolDetails);
```

Comment:
is it required to sent these details in ModifyStoragePoolAnswer ?

Reply:
I have removed using the ModifyStoragePool command